infra/base-images/base-runner/coverage

```diff
@@ -136,6 +149,12 @@ for fuzz_target in $FUZZ_TARGETS; do
   done
 done
 
+if [[ $GO_TARGET == "true" ]]; then
+  echo "End for go targets"
+#TODO merge all fuzz targets reports cf https://github.com/wadey/gocovmerge
+  exit 0
```

Comment:
If I get it right, the logic for C projects is to run the fuzz targets, and then merge coverage profiles.
We can do this for golang, it seems there is a project that does it with 111 lines

Or we can, from the start, have the command `go test -v -coverprofile` running all the fuzz targets